src/hotspot/share/gc/g1/g1CardSet.cpp

```diff
@@ -34,6 +34,7 @@
 #include "runtime/globals_extension.hpp"
 #include "runtime/java.hpp"
 #include "utilities/bitMap.inline.hpp"
+#include "utilities/concurrentHashTable.hpp"
```

Comment:
Do you needs this? Doesn't concurrentHashTable.inline.hpp include it already?

Reply:
Fixed :).